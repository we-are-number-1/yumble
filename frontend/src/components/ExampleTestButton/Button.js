import React from 'react';

export default function Button({text}) {
    return (
        <div>
            <button>{text}</button>
        </div>
    );
}