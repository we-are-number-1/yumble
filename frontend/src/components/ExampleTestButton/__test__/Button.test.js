import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../Button';

test('Button snapshot testing', () => {
  const component = renderer.create(
      <Button text="Testing Button"></Button>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
