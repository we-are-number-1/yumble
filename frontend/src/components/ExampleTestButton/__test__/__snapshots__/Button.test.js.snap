// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot testing 1`] = `
<div>
  <button>
    Testing Button
  </button>
</div>
`;
